src/rpc/connect.py

```diff
@@ -0,0 +1,147 @@
+"""
+Rpc connect
+"""
+
+from __future__ import annotations
+import threading
+import asyncio
+import grpc
+import curp_command_pb2
+import curp_command_pb2_grpc
+
+
+class Connect:
+    """
+    The connection struct to hold the real rpc connections,
+    it may failed to Connect, but it also retries the next time
+    """
+
+    # Server serverId
+    __server_id: int
+    # The rpc connection
+    __rpc_connect: list[grpc.Channel]
```

Comment:
There is any load banlancer libs ?

Reply:
It will be implemented in the subsequent pr，I added a TODO and Issue to keep track of.